cypress/support/commands.js

```diff
@@ -33,3 +35,48 @@ Cypress.Commands.add("openRegisterDialogue", () => {
     cy.get('.dropdown-toggle').contains('LOGIN').click();
     cy.get('a.btn-block').contains('Create new account').click();
 })
+
+Cypress.Commands.add("openUploadTab", () => {
+    cy.get("#bluegenes-main-nav").within(() => {
+        cy.contains("Upload").click();
+    });
+})
+
+Cypress.Commands.add("openListsTab", () => {
+    cy.get("#bluegenes-main-nav").within(() => {
+        cy.contains("Lists").click();
+    });
+})
+
+Cypress.Commands.add("openTemplatesTab", () => {
+    cy.get("#bluegenes-main-nav").within(() => {
+        cy.contains("Templates").click();
+    });
+})
+
+Cypress.Commands.add("openRegionsTab", () => {
+    cy.get("#bluegenes-main-nav").within(() => {
+        cy.contains("Regions").click();
+    });
+})
+
+Cypress.Commands.add("openQueryBuilderTab", () => {
+    cy.get("#bluegenes-main-nav").within(() => {
+        cy.contains("Query Builder").click();
+    });
+})
+
+Cypress.Commands.add("searchKeyword", (keyword) => {
+    cy.get(".search").first().type(keyword + '{enter}',{delay:100});
```

Comment:
For `Search Test can search two terms with OR`, it failed 3 times to type in *CDPK1 OR CDPK4* correctly. Instead of increasing the delay, what do you think about using `cy.visit("/biotestmine/search");` in the beforeEach for these tests, and then using the search input in the page instead of the one in the navbar? This one won't return suggested results, so it should be less likely to fail in CI.